include/arch/arm/arch/32/mode/machine/timer.h

```diff
@@ -16,7 +16,7 @@
  * CLK_MAGIC and TIMER_CLOCK_MHZ -- these definitions might need to move
  * if we come across an arm platform that does not suit this model */
 
-/* get the max value ticksToUs can be passed without overflowing */
+/* Get the max value ticksToUs() can handle without overflowing. */
```

Comment:
this change is a bit pedantic - what's wrong with 'be passed`? and I don't want to encourage rewriting all of the comments to use correct capitalisation - too hard to enforce, and way too little benefit to bother with.